packages/kbn-es-query/src/kuery/utils/escape_kuery.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
@elastic/kibana-data-discovery team, 

I have ported this from below file to merged to `7.17`. https://github.com/elastic/kibana/blob/9fb17237953f9b1444d76436c8b20c9a1523ee36/x-pack/plugins/timelines/public/components/utils/keury/index.test.ts#L53-L59

Reply:
This file actually exists in 7.17, it's just in a different location: https://github.com/elastic/kibana/blob/7.17/src/plugins/data/public/autocomplete/providers/kql_query_suggestion/lib/escape_kuery.ts

It looks like it was moved in https://github.com/elastic/kibana/pull/128289/files. Maybe we can remove the other one in this PR so we don't have a duplicate?

Reply:
Thanks @lukasolson. I have deleted the duplicate and reused the one existing here: 82d8fed076b